pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+# SAMPL6 pKa Challenge Instructions
+
+Challenge timeframe: Oct 24, 2017 to Jan 10, 2018  
+
+This challenge consists of predicting microscopic and macroscopic pKas of 24 small organic molecules. These fragment-like small molecules are selected for their similarity to kinase inhibitors and for experimental tractability. Our aim is to evaluate how well current pKa prediction methods perform with drug fragment-like molecules through blind predictions.
+
+Three formats of pKa prediction results will be evaluated:
+1. microscopic pKa values and related microstates
+2. microstate populations as a function of pH
+3. macroscopic pKa values
```

Comment:
I feel like somewhere in this file we need to define what we mean by microscopic and macroscopic pKa. It might feel redundant, but I'm not sure if everyone uses the same vocabulary for each of these measurements.

Reply:
Yes, I had that thought too.

Reply:
Done.